acceleration/main.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"math/rand"
+	"net/http"
+	"os"
+
+	"github.com/spiceai/sample-1/pkg/flightsql"
+
+	_ "github.com/lib/pq"
+)
+
+var PRODUCTS = []string{"the most beautiful flower", "the most delicious chocalate", "nintendo"}
+
+func main() {
+	fmt.Println("Start the exmaple application to use Spice")
```

Comment:
```suggestion
	fmt.Println("Start the example application to use Spice")
```

Reply:
Please resolve / address in next iteration.  Closing comment to merge.